oneview_redfish_toolkit/blueprints/zone.py

```diff
@@ -68,6 +71,13 @@ def get_zone(zone_uuid):
     zone_data = Zone(zone_uuid, profile_template, server_hardware_list,
                      enclosure_name, drives)
 
+    sh_count = len(server_hardware_list)
+    blocks_count = len(zone_data.redfish["Links"]["ResourceBlocks"])
+    logging_service.debug(COUNTER_LOGGER_NAME,
+                          str(len(drives)) + " Drives retrieved",
```

Comment:
Maybe we could change the format to keep the counter as the last information:
2018-09-18 15:20:17,362 - counter - DEBUG - Thread 139789332358912: Enclosures retrieved: 3
2018-09-18 15:20:17,462 - counter - DEBUG - Thread 139789332358912: ServerHardware retrieved: 7